WooCommerce/Classes/ViewRelated/Dashboard/Settings/In-Person Payments/Onboarding Errors/InPersonPaymentsOnboardingErrorMainContentView.swift

```diff
@@ -0,0 +1,53 @@
+import SwiftUI
+import Yosemite
+
+struct InPersonPaymentsOnboardingErrorMainContentView: View {
+    let title: String
+    let message: String
+    let image: ImageInfo
+    let supportLink: Bool
+
+    struct ImageInfo {
+        let image: UIImage
+        let height: CGFloat
+    }
+
+    @Environment(\.verticalSizeClass) var verticalSizeClass
+
+    var isCompact: Bool {
+        get {
+            verticalSizeClass == .compact
+        }
+    }
+
+    var body: some View {
+        VStack(alignment: .center) {
+            Text(title)
+                .font(.headline)
+                .padding(.bottom, isCompact ? 16 : 32)
+            Image(uiImage: image.image)
+                .resizable()
+                .scaledToFit()
+                .frame(height: isCompact ? image.height / 3 : image.height)
+                .padding(.bottom, isCompact ? 16 : 32)
```

Comment:
I wonder if if a way to avoid complication when we're checking this View on landscape, could be simply to remove the Image render via an `!isCompact` check:

![Simulator Screen Shot - iPhone 11 - 2022-08-17 at 13 21 15](https://user-images.githubusercontent.com/3812076/185106609-3b9663e7-3266-4ce3-9644-aa49144f9118.png)

Or if we want the image, possibly move to a Zstack or HStack-based design.

Reply:
That works, but will still be quickly defeated by a larger font. It's already truncating the message, after all.